lambda-resume-builder/src/Components/Calendar/Calendar.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const Calendar = () => {
+    
+        return (
+            <h1>This Is The Calendar Checklist Component</h1>
+        )
+    
+        
+    
+    
```

Comment:
### I really don't understand why we don't have an actual Calendar component done yet.

If you're looking for inspiration maybe look into doing something along the lines of [this](https://imgur.com/lyqHIva). That calendar seems like a much better choice than a simple `<h1>`, and like it might be able to win awards.